bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/URLAudioStream.java

```diff
@@ -90,7 +90,10 @@ private InputStream createInputStream() throws AudioException {
                 os.write(req.getBytes());
                 is = shoutCastSocket.getInputStream();
             } else {
-                is = streamUrl.openStream();
+                // getInputStream() method is more error-proof than openStream(),
+                // because openStream() does openConnection().getInputStream(), 
+                // which opens a new connection and does not reuse the old one.
+                is = connection.getInputStream();
```

Comment:
> I put it in this PR, because it is a small change, but if you want I can make a separate PR with this change.

You should do it, not because I want it, but because that fix is not only related to the test.

Let's assume there is something completely wrong with this test (I don't think so, but ...), if we revert the commit that introduced that test, also the bugfix will be removed.
Let's assume someone is interested why the change has been done (without following every discussion on Github) and uses `git blame` and the Git history as a source of information (as it is intended by that VCS). Then the change is marked to be done because of testing. That is slightly not correct.